tests/test_client.py

```diff
@@ -172,6 +174,7 @@ def test_audiotranscription_ok():
     task = client.create_audiotranscription_task(
         callback_url='http://www.example.com/callback',
         attachment_type='audio',
+        instruction='Listen the audio file and transcript.',
```

Comment:
We don't really do audio transcription - in 2.0, let's just remove this task.

Small spelling error here as well, missing "Listen to"

Reply:
Fixed!